src/ServerConfigParams.ts

```diff
@@ -1,191 +1,197 @@
-import parseArgs from 'minimist';
-import * as fs from 'fs';
+import { configure } from '@rsksmart/rif-relay-client';
 import {
-    VersionRegistry,
-    ContractInteractor,
-    constants
+  constants,
+  ContractInteractor,
+  VersionRegistry
 } from '@rsksmart/rif-relay-common';
-import { configure } from '@rsksmart/rif-relay-client';
+import * as fs from 'fs';
+import parseArgs from 'minimist';
 import { KeyManager } from './KeyManager';
 import { TxStoreManager } from './TxStoreManager';
 
 //@ts-ignore
-import sourceMapSupport from 'source-map-support';
-//@ts-ignore
-sourceMapSupport.install({ errorFormatterForce: true });
-import { LogLevelNumbers } from 'loglevel';
-import log from 'loglevel';
 import 'dotenv/config';
+import log, { LogLevelNumbers } from 'loglevel';
+// TODO: had to comment out as it doesn't work. Maybe let's update the dependency ;]
+// import sourceMapSupport from 'source-map-support';
+//@ts-ignore
+// sourceMapSupport.install({ errorFormatterForce: true });
 
 // TODO: is there a way to merge the typescript definition ServerConfigParams with the runtime checking ConfigParamTypes ?
+// FIXME: Yes there is with Pattern Matching. Let's create a ticket for refactoring this. We could also move to node config which would make life easier I suspect.
```

Comment:
```suggestion
// FIXME: Yes there is with Pattern Matching. Let's create a ticket for refactoring this. We could also move to node config which would make life easier I suspect, see [PP-314](https://rsklabs.atlassian.net/browse/PP-314).
```
Thank you for the suggestion, I created a task for this, feel free to update the description if it's not clear enough.

Reply:
done